recipe/meta.yaml

```diff
@@ -1,23 +1,23 @@
-{% set version = "2.2.4" %}
+{% set version = "2.3.0" %}
 
 package:
   name: libgdal
   version: {{ version }}
 
 source:
   url: http://download.osgeo.org/gdal/{{ version }}/gdal-{{ version }}.tar.xz
-  sha256: 441eb1d1acb35238ca43a1a0a649493fc91fdcbab231d0747e9d462eea192278
+  sha256: 6f75e49aa30de140525ccb58688667efe3a2d770576feb7fbc91023b7f552aa2
   patches:
     # Fixes compilation problems with JPEG.
     # https://lists.osgeo.org/pipermail/gdal-dev/2016-January/043443.html
-    - disable_jpeg12.patch
+    # - disable_jpeg12.patch
     # BUILT_AS_DYNAMIC_LIB.
-    - windowshdf5.patch
+    # - windowshdf5.patch
     # Use multiple cores on Windows.
-    - multiprocessor.patch
+    # - multiprocessor.patch
```

Comment:
These two may need to be refreshed. Can you confirm that @gillins?
(I can do that later over the weekend.)